pkg/ottl/ottlfuncs/README.md

```diff
@@ -1285,7 +1286,70 @@ Examples:
 
 - `ParseXML("<HostInfo hostname=\"example.com\" zone=\"east-1\" cloudprovider=\"aws\" />")`
 
+### RemoveXML
 
+`RemoveXML(target, xpath)`
```

Comment:
Since this function returns an error if the string is not parsable xml, can we include something to help users write a where clause that checks this requirement?

For `ParseJSON` users typically do `where IsMatch(body, "\\{")`. What should they do for xml?

Reply:
`where IsMatch(body, "\\<")` should work about as reliably for xml as the other does for json. I'm not sure how robust either is really since there can of course be other strings which start with these characters. I guess I would slightly prefer to leave these undocumented for now and we can always add `IsJSON` and/or `IsXML` functions later if necessary.

Reply:
As long as there is some sort of solution that xml users will understand I'm good with that 

My gut says that IsJSON or IsXML will be inefficient, so. I wonder if the decision to error in ParseJSON/XML is correct.  We should stick with it for now and can address later.